client/src/store/reducer/createSurveyReducer.js

```diff
@@ -17,6 +17,7 @@ const initalState = {
   disclaimer: 'This is the dummy disclaimer',
   anonymous: false,
   questions: [defaultQuestion],
+  dateCreated: Date.now(),
```

Comment:
Please make this a UTC string

Reply:
...or convince me it should be an integer

Reply:
UNIX integers i think are more interchangeable and are a more simple format to work with.